### PR TITLE
Add "getAccountDetails" base implementation

### PR DESCRIPTION
**Pull request by jkuptz-ba**
The UniCredit Group returns the `ownerName` exclusively in the GET response for the "Read Account Details" endpoint.

This pull request contains the base implementation for the "Read Account Details" endpoint.

### CODE REVIEW

**Comment by berezkin88** [comment]
`ownerName` is a part of Berlin Group spec for some time

I've added some amendments to your PR on your fork, please check https://github.com/BANKSapi/xs2a-adapter/pull/2

Please update your branch, it will be merged after

File: adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java [hunk 1]
```diff
@@ -60,6 +60,13 @@ public UnicreditAccountInformationService(Aspsp aspsp,
         return super.getAccountList(requestHeaders, RequestParams.builder().build());
     }
 
+    @Override
+    public Response<OK200AccountDetails> getAccountDetails(String accountId,
+                                                           RequestHeaders requestHeaders,
+                                                           RequestParams requestParams) {
+        return super.getAccountDetails(accountId, requestHeaders, RequestParams.builder().build());
```
Comment (berezkin88) [inline adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java:67]: Why RequestParams should be reset here?

Comment (jkuptz-ba) [reply adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java]: I implemented it analogously to "Read Account List". The only query param for these endpoints is "withBalance".

According to the UniCredit API documentation this query param will be ignored for "Read Account List" and "Read Account Details".

I just did a test with the query param "withBalance" (true and false) and got no error for "Read Account List" and "Read Account Details". I think it is not necessary to reset the RequestParams for both endpoints.

**Comment by jkuptz-ba** [comment]
Thanks for your PR. I merged it into our fork.

File: adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java (hunk 1 above)
Comment (berezkin88) [reply adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java]: Good point!
`readAccountList` shouldn't have a RequestParams reset, it was implemented that way with no clear reason and will be fixed in the future.

I'd recommend not overriding `readAccountDetails` here and omit passing empty RequestParams

Also, please update this branch with the latest Adapter commits from the develop one, you're missing some stuff

Comment (jkuptz-ba) [reply adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java]: OK :+1: I removed this method and updated our fork.

**Review by berezkin88 — APPROVED** [review]

**Pull request merged**